python/pyspark/sql/dataframe.py

```diff
@@ -296,8 +312,10 @@ def show(self, n=20, truncate=True):
             If set to a number greater than one, truncates long strings to length ``truncate``
             and align cells right.
 
+        >>> from pyspark.sql import Row
+        >>> df = spark.createDataFrame([Row(name='Alice', age=2), Row(name='Bob', age=5)])
         >>> df
-        DataFrame[age: int, name: string]
+        DataFrame[age: bigint, name: string]
```

Comment:
This is the same. It seems `bigint` by default if we don't explicitly provide the schema.